### PR TITLE
feat: add prusa-connect-rtsp

### PR DESCRIPTION
**Pull request by beccazero**

### CODE REVIEW

File: templates/prusa-connect-rtsp.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <Support/>
+  <Project>https://github.com/beccazero/prusa-connect-rtsp</Project>
+  <Overview>Capture images from a RTSP camera upload them to Prusa Connect.</Overview>
+  <Category>HomeAutomation:</Category>
+  <WebUI/>
+  <TemplateURL/>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/prusa-connect-rtsp.png</Icon>
+  <ExtraParams/>
+  <PostArgs/>
+  <CPUset/>
```
Comment (nwithan8) [inline templates/prusa-connect-rtsp.xml:19]: There's a lot of unused variables in here that got auto-added by the plugin you likely used to convert an installed Docker container into an Unraid template. Please clean them up :)

**Review by nwithan8 — CHANGES_REQUESTED** [review]

**Comment by beccazero** [comment]
Removed the unused variables.

File: templates/prusa-connect-rtsp.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  <Project>https://github.com/beccazero/prusa-connect-rtsp</Project>
+  <Overview>Capture images from a RTSP camera upload them to Prusa Connect.</Overview>
+  <Category>HomeAutomation:</Category>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/prusa-connect-rtsp.png</Icon>
+  <DateInstalled>1703979411</DateInstalled>
+  <Description>Capture images from a RTSP camera upload them to Prusa Connect.</Description>
+  <Networking>
+    <Mode>bridge</Mode>
+  </Networking>
+  <Environment>
```
Comment (nwithan8) [inline templates/prusa-connect-rtsp.xml:17]: Remove this Environment section.

File: templates/prusa-connect-rtsp.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <Name>prusa-connect-rtsp</Name>
+  <Repository>ghcr.io/beccazero/prusa-connect-rtsp:main</Repository>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Project>https://github.com/beccazero/prusa-connect-rtsp</Project>
+  <Overview>Capture images from a RTSP camera upload them to Prusa Connect.</Overview>
+  <Category>HomeAutomation:</Category>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/prusa-connect-rtsp.png</Icon>
+  <DateInstalled>1703979411</DateInstalled>
```
Comment (nwithan8) [inline templates/prusa-connect-rtsp.xml:12]: Remove this

**Review by nwithan8 — CHANGES_REQUESTED** [review]

**Comment by Roxedus** [comment]
LGTM after the requested changes.

**Comment by nwithan8** [comment]
Closing due to inactivity

**Pull request closed without merging**